tools/analysis/analysis_runner.py

```diff
@@ -0,0 +1,77 @@
+# Analysis runner.
+import argparse
+import os
+import subprocess
+
+# TODO: Support customizing these.
+_TOPIC = 'projects/ossf-malware-analysis/topics/workers'
+_PACKAGES_BUCKET = 'ossf-malware-analysis-packages'
+
+
+def _upload_file(local_path):
+  # TODO: figure out a better way to key these.
+  filename = os.path.basename(local_path)
+  upload_path = f'gs://{_PACKAGES_BUCKET}/{filename}'
+  print('Uploading', local_path, 'to', upload_path)
+  subprocess.run(
+    ('gsutil', 'cp', local_path, upload_path), check=True)
```

Comment:
Can `gsutil` be listed as a dep somewhere.

Reply:
This is all part of Google Cloud SDK. I've added this to the README.